src/devices/Device-Index.md

```diff
@@ -32,6 +32,7 @@
 * [Charlieplex Segment binding](Charlieplex/README.md)
 * [Cpu Temperature](CpuTemperature/README.md)
 * [DC Motor Controller](DCMotor/README.md)
+* [DFRobot KIT0176: I2C 1kg Weight Sensor Kit - HX711 (Gravity: I2CWeight Sensor)](Hx711I2c/README.md)
```

Comment:
while regenerating this file, what about going to the title of the README and adjusting them to add more about the device names? Like this one: https://github.com/dotnet/iot/blob/main/src/devices/Bmxx80/README.md
Would be great to list of the devices.

Reply:
I've created https://github.com/dotnet/iot/issues/1967 to track all comments you've put here but unfortunately I'd prefer this one was scoped to simple change. I was thinking about doing this before I opened PR but realized I'm a bit short on time so decided to just do simple change